src/main/java/de/hysky/skyblocker/skyblock/waypoint/MythologicalRitual.java

```diff
@@ -152,6 +212,9 @@ public static void render(WorldRenderContext context) {
                     if (burrow.echoBurrowLine != null) {
                         RenderHelper.renderLinesFromPoints(context, burrow.echoBurrowLine, ORANGE_COLOR_COMPONENTS, 0.5F, 5F, false);
                     }
+                    if (burrow.nextBurrowEstimatedPos != null && !burrow.shouldRender()) {
```

Comment:
Maybe `burrow.confirmed == TriState.DEFAULT` is easier to understand than `!burrow.shouldRender()`? Actually, is `!burrow.shouldRender()` even needed?